### PR TITLE
Update permissions.js and ignoreMessagesBeforeStart.js

### PR DESCRIPTION
**Pull request by YouTwitFaceTG**

### CODE REVIEW

File: plugins/permissions.js [hunk 1]
```diff
@@ -2,17 +2,24 @@ module.exports = {
...
-                data.message = {};
-                bot.sendMessage(userId, pluginConfig.message);
+            if (data.message.chat.type === 'channel') {
+                const chatId = data.message.chat.id;
+                bot.sendMessage(userId, pluginConfig.message).then(() => {
+                    bot.leaveChat(chatId);
+                });
+            } else {
+                const userId = data.message.from.id;
+                if (pluginConfig.allowedUserNames.includes(userId) < 0) {
```
Comment (Shemesh) [inline plugins/permissions.js]: allowedUserNames should be allowedUserIds

**Comment by YouTwitFaceTG** [comment]
I already fixed that in the the 3rd commit

File: plugins/ignoreMessagesBeforeStart.js [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
-    plugin(bot) {
+    defaultConfig: {
+        start: true
+    },
 
-        bot.on(['start', '/start'], () => {
-            startDate = Date.now();
+    plugin(bot, config) {
+
+        bot.on([config.start ? 'start', '/start' : 'start'], (msg) => {
```
Comment (mullwar) [inline plugins/ignoreMessagesBeforeStart.js]: Error on this line, please fix.

Comment (YouTwitFaceTG) [reply plugins/ignoreMessagesBeforeStart.js]: Whats the error?

Comment (mullwar) [reply plugins/ignoreMessagesBeforeStart.js]: /home/travis/build/mullwar/telebot/plugins/ignoreMessagesBeforeStart.js
  13:39  error  Parsing error: Unexpected token ,

**Comment by YouTwitFaceTG** [comment]
#129

**Pull request merged**